Resources/App_GlobalResources/SharedResources.nl-NL.resx

```diff
@@ -1384,7 +1354,7 @@
     <value>Publiek</value>
   </data>
   <data name="PublicMembership.Text" xml:space="preserve">
-    <value>&lt;strong&gt;* Opmerking: &lt;/strong&gt; Lidmaatschap van deze website is publiek. Nadat je accountinformatie is verzonden, heb je onmiddellijk toegang tot de website.</value>
+    <value>&lt;strong&gt;* Opmerking: &lt;/strong&gt; Lidmaatschap van deze website is publiek. Als je account informatie is verzonden, dan heb je onmiddelijk toegang tot de website.</value>
```

Comment:
Het lidmaatschap van deze website is publiek. Als je account informatie is verzonden, dan heb je onmiddellijk toegang tot de website.